OffchainLabs/arbitrum-token-bridge
# fix: clear stale pending tx

### Steps to test


1. on production, edit `arbTransactions` in your local storage to have a tx created >7 days ago with the "pending" status
2. note that you see the pending tx card on tx history panel
3. on this branch, use the same `arbTransactions`
4. note that the pending tx card should not exist on the tx history panel

If some pending txs are created <7 days, the stale pending txs will still stay.
This PR only implements clearing if all pending txs are created >7 days.

Closes #837
